comixed-state/src/main/java/org/comixedproject/state/comic/ComicStateMachineConfig.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * ComiXed - A digital comic book library management application.
+ * Copyright (C) 2021, The ComiXed Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses>
+ */
+
+package org.comixedproject.state.comic;
+
+import java.util.EnumSet;
+import org.comixedproject.model.comic.ComicState;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.statemachine.config.EnableStateMachine;
+import org.springframework.statemachine.config.EnumStateMachineConfigurerAdapter;
+import org.springframework.statemachine.config.builders.StateMachineStateConfigurer;
+import org.springframework.statemachine.config.builders.StateMachineTransitionConfigurer;
+
+/**
+ * <code>ComicStateMachineConfig</code> provides a state machine configuration to manage the state
+ * for instances of {@link Comic}.
```

Comment:
Exit code: 1 - /home/runner/work/comixed/comixed/comixed-state/src/main/java/org/comixedproject/state/comic/ComicStateMachineConfig.java:31: error: reference not found
Error:   * for instances of {@link Comic}.

-> Import missing : import org.comixedproject.model.comic.Comic;

Reply:
Fixed.